test/e2e/ha/openstack_services_test.go

```diff
@@ -0,0 +1,226 @@
+package ha
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	contrail "github.com/Juniper/contrail-operator/pkg/apis/contrail/v1alpha1"
+	"github.com/Juniper/contrail-operator/pkg/controller/utils"
+	"github.com/Juniper/contrail-operator/test/logger"
+	"github.com/Juniper/contrail-operator/test/wait"
+	"github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/operator-framework/operator-sdk/pkg/test/e2eutil"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	core "k8s.io/api/core/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+func TestHAOpenStackServices(t *testing.T) {
+	ctx := test.NewTestCtx(t)
+	defer ctx.Cleanup()
+	log := logger.New(t, "contrail", test.Global.Client)
```

Comment:
can this logger be called differently, like "openstack-test" ?

Reply:
not really since this actually a name of the namespace

Reply:
I will move its creation to use namespace variable

Reply:
oh, ok